packages/opentelemetry-propagator-jaeger/src/JaegerHttpTraceFormat.ts

```diff
@@ -86,7 +86,8 @@ export class JaegerHttpTraceFormat implements HttpTextFormat {
  * @return {SpanContext} - returns a span context represented by the serializedString.
  **/
 function deserializeSpanContext(serializedString: string): SpanContext | null {
-  let headers = serializedString.split(':');
+  let decoded = decodeURIComponent(serializedString);
```

Comment:
Please use const when variables are not reassigned.

```typescript
const headers = decodeURIComponent(serializedString).split(':');
```

Reply:
Done, used variable because thought it more clearly